vrx_gazebo/worlds/xacros/wildlife.xacro

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<world xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:macro name="wildlife" params="**animal_includes
+                                       **animal_buoys
+                                       namespace:=wamv
+                                       competition:=vrx">
+    <xacro:insert_block name="animal_includes"/>
+    <!-- The scoring plugin -->
+    <plugin name="wildlife_scoring_plugin"
+            filename="libwildlife_scoring_plugin.so">
+      <!-- Common parameters -->
+      <vehicle>${namespace}</vehicle>
+      <task_name>wildlife_encounter</task_name>
```

Comment:
According to the [task description document](https://robonation.org/app/uploads/sites/2/2021/09/VirtualRobotX2022_Task-Descriptions_v1.0.pdf), the name of this task should be `wildlife`.

Reply:
I took the liberty of updating the task names for the "station keeping" and "wildlife encounter" tasks. See 5131a0f.

Reply:
Thanks for fixing it!